src/router/index.js

```diff
@@ -136,6 +136,12 @@ const routes = [
     component: () => import('../pages/RegisterStudents.vue'),
     meta: { pageTitle: 'Register Students', requireAdmin: true, requireSuperAdmin: true },
   },
+  {
+    path: '/register-users',
+    name: 'Register Users',
+    component: () => import('../pages/LEVANTE/RegisterUsers.vue'),
```

Comment:
We discussed implementing project-specific route guardsbased on the env variable, so that the projects would not have access to outside routes easily.